SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
+
+1.  [Install Haskell][1]. While you are at it, make sure you have the
+    `base-devel` group installed.
+2.  [Install PostgreSQL][2]. You don't need to go through setting up a
+    user or anything like that. Snowdrift will do that for you.
+3.  Run these commands:
+
+        git clone https://git.gnu.io/snowdrift/snowdrift.git
+        cd snowdrift
+        ln -s cabal.config.7.10 cabal.config
+        cabal sandbox init
+        cabal install -j -fdev --enable-tests
```

Comment:
It was reported by @singpolyma that running `--enable-tests` failed until *after* the `sdm init` was done. That makes little *sense* to me, but I'm taking his word for it, haven't tested myself. Thus, we changed the GUIDE and the SETUP_DEBIAN files to run without the enable-tests first and then do --enable-tests *after* the sdm init (we can't just reverse these because the install command has to build sdm!)

So, either confirm that he is wrong and this isn't needed (and maybe we should adjust the other files), or make the same fix here as in the other files. Either way, this aspect of the instructions should be consistent.